### DISCUSSION TITLE
Ensure that the [name, id, data] of the Livewire component wasn't tampered with between requests.

### DISCUSSION THREAD

**Discussion by F3nrihr** [abordage/laravel-html-min#7] [Q&A]
Hello.
In production if I active HTML_MINIFY I has liveware error 500:
![image](https://user-images.githubusercontent.com/18676406/199563732-90b38668-85b2-435e-b835-a1597788d0ba.png)
It is surprising because in the staging deploy it works perfectly.

I use vapor and in the deploy it only changes

npm ci && npm run dev && rm -rf node_modules for npm ci && npm run prod && rm -rf node_modules

Thanks for your time

**Comment by abordage** [comment]
Hi, @F3nrihr
Try turning off middleware GzipEncodeResponse or/and Laravel Page Speed

**Reply by F3nrihr** [reply]
Hello @abordage .
I uninstalled laravel page speed and I installed laravel-html-min.
If I leave Gzip alone, it works, that's how I have it currently in production because otherwise liveware has that 500 error.
It is curious because in stagin it works perfectly.
